packages/common/src/ui/components/popover/BasePopover/BasePopover.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { FC } from 'react';
+import Fade from '@mui/material/Fade';
+import Paper, { PaperProps } from '@mui/material/Paper';
+import Popper, { PopperProps } from '@mui/material/Popper';
+
+export interface BasePopoverProps extends Omit<PopperProps, 'open'> {
+  // I like having is as a prefix for boolean props
+  isOpen: boolean;
+  anchorEl: PopperProps['anchorEl'];
+  paperProps?: PaperProps;
+}
+
+export const BasePopover: FC<BasePopoverProps> = ({
```

Comment:
I don't know if this is a good name - I can see there being more specific variants - `DetailPopover` and could (should?) be for the `ColorMenu`.